<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>flubberpp demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGraphicsView" name="view">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="5">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="4">
          <widget class="QToolButton" name="categoryBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="media-skip-forward">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="categoryBtn_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="media-skip-backward">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="categoryLbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>One to One</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Color</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="colorCombo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yellow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cyan</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Purple</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QToolButton" name="prevBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="media-seek-backward">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="nextBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="media-seek-forward">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="playBtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset theme="media-playback-pause">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="5">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1" colspan="2">
            <widget class="QLabel" name="timeLbl">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>dt</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QSlider" name="timeSlider">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>prevBtn</tabstop>
  <tabstop>nextBtn</tabstop>
  <tabstop>playBtn</tabstop>
  <tabstop>timeSlider</tabstop>
  <tabstop>colorCombo</tabstop>
  <tabstop>categoryBtn_2</tabstop>
  <tabstop>categoryBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
